README.md

```diff
@@ -5,46 +5,29 @@
 -->
```

Comment:
Just a general comment, since we have now adopted the new naming conventions for boray, electric boray etc. can we also change them here?

Reply:
Not as part of this, no. We'll do that in a subsequent separate PR so as not to conflate two issues.

Reply:
Got it and it makes sense too.